packages/notification-services-controller/package.json

```diff
@@ -128,7 +128,7 @@
   },
   "peerDependencies": {
     "@metamask/keyring-controller": "^19.0.0",
-    "@metamask/profile-sync-controller": "^3.0.0"
```

Comment:
It looks like we are bumping a peer dependency here. If we were to bump `profile-sync-controller` in a client that also uses `notification-services-controller` then it would give us a peer dep warning since `notification-services-controller` expects v3 but we now are using v4. 

Should we include this package in the release too (and bump it by a major)?